svg/attributes/data.json

```
@@ -0,0 +1,54 @@
+{
+  "svg": {
+    "attributes": {
+      "data": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Attribute/data-*",
+          "support": {
+            "chrome": {
+              "version_added": "55"
+            },
+            "chrome_android": {
+              "version_added": "55"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "51"
+            },
+            "firefox_android": {
+              "version_added": "51"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": "41"
```

I can't find anything explicitly about the attributes themselves, but this bug makes me think that Opera 42 is more likely than 41 https://bugs.chromium.org/p/chromium/issues/detail?id=636108

```suggestion
              "version_added": "42"
```

---

42 looks correct to me as well, as it follows the Chromium-13 policy of Opera’s version numbers!